debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/Filters.java

```diff
@@ -30,49 +27,53 @@
 
     protected static final Set<String> BUILT_IN_DB_NAMES = Collect.unmodifiableSet("mysql", "performance_schema", "sys", "information_schema");
 
+    /**
+     * A list of tables that are always ignored. Useful for ignoring "phantom"
+     * tables occasionally exposed by services such as Amazon RDS Aurora. See
+     * DBZ-1939.
+     */
+    private static final Set<String> IGNORED_TABLE_NAMES = Collect.unmodifiableSet(
+            "mysql.rds_configuration",
+            "mysql.rds_global_status_history",
+            "mysql.rds_global_status_history_old",
+            "mysql.rds_history",
+            "mysql.rds_replication_status",
+            "mysql.rds_sysinfo");
```

Comment:
Maybe we should later make this a config property (might be in RelationalDatabaseConnectorConfig) that gets sane defaults by different connector implementations (mysql will set this default list, PG a different one).

Or one step ahead, what about configuring "database.environment" where users can select things like amazon_rds_mysql, amazon_rds_postgres, standard_mysql, standard_postgres_huge_transactions, etc and we pick "the best defaults" and correct builtin tables? might make some people's life easier?

Reply:
Perhaps, but I don't think it's needed. Let's not overcomplicate things by adding too many knobs and options. Also keep in mind that the issue supposedly is fixed by AWS (in some cases?) already.